tangelo/problem_decomposition/electron_localization/nao_localization.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2021 Good Chemistry Company.
```

Comment:
If defining this function makes sense for you I'm all for it.

I just want us to acknowledge that currently we have a new file with 38 lines of code to maintain, for a function call that encapsulate 1 import and call a function. It is not a bad thing as long as it makes sense from a user perspective, and we are actually using it.